content/docs/command-reference/freeze.md

```diff
@@ -77,12 +77,4 @@ Run `dvc unfreeze` to unfreeze it back:
 
 ```dvc
 $ dvc unfreeze make_copy
```

Comment:
https://github.com/iterative/dvc.org/pull/1488/files#r444374258

I think there is no need to put the statement in the end, as it is stated in the beginning:
"Run dvc unfreeze to unfreeze it back."